generator-core/src/main/java/br/com/generator/core/GeneratorExecutor.java

```diff
@@ -35,7 +35,7 @@ public File addDependecies(GeneratorOptions options) throws IOException {
         File fileGenerated = new File(options.getDestination());
         FileUtils.writeStringToFile(fileGenerated, contentReplaced);
```

Comment:
Consider using a try-with-resources statement to ensure `FileUtils.writeStringToFile` handles potential exceptions and closes resources.